packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -39,6 +39,7 @@ function MapsIndoorsMap({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId
     const [currentLocation, setCurrentLocation] = useState();
     const [currentCategories, setCurrentCategories] = useState([]);
     const [filteredLocations, setFilteredLocations] = useState();
+    const [initialFilteredLocations, setInitialFilteredLocations] = useState();
```

Comment:
Write some comments so it will be easier to differentiate between filteredLocations and initialFilteredLocations.

Reply:
✅